examples/example_parameters.param

```diff
@@ -0,0 +1,106 @@
+#############
+### Input ###
+#############
+# The path and naming pattern of the input image series.
+# For instance, a valid name pattern is "/Volumes/Data/Cpp/input/StEM/ADF%04d.dm3".
+# The frame counter is encoded in "printf" style, e.g. `%04d` means here is the counter, it has four digits and unused digits are filled with zeros, e.g. ADF0000, ADF0001, ADF0002, ...
+# When using the python wrapper this will be filled out automatically
+templateNamePattern " "
+# The first frame to start the calculation.
+# If 10, then in the above example the first frame would be ADF0010
+templateNumOffset 0
+# The step between subsequent images. In the above example, if 2 then images are ADF0000, ADF0002, ADF0004, ADF0006, ...
+templateNumStep 1
+# Maximum number of images to consider.
+numTemplates 0
```

Comment:
2 is the smallest reasonable setting for `numTemplates`, so perhaps we should set the default to 2.